components/cli/cmd/cellery/delete_image.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/cellery-io/sdk/components/cli/pkg/commands"
+)
+
+func newDeleteImageCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete <cell-image(s)>",
+		Short: "Delete cell image(s) from repo",
+		Args: func(cmd *cobra.Command, args []string) error {
+			err := cobra.ExactArgs(1)(cmd, args)
+			if err != nil {
+				return err
+			}
+			return nil
+		},
+		Run: func(cmd *cobra.Command, args []string) {
+			commands.RunDeleteImage(args[0])
+		},
+		Example: "  cellery delete cellery-samples/employee:1.0.0",
```

Comment:
Do we need this leading space?
Also, shall we add a sample with regex?

Reply:
* Space is needed to make the help text in CLI more clearer to the user.
* Updated and added 3 exampled including regex.